src/components/TopBar.js

```diff
@@ -1,4 +1,4 @@
-import logo from './logo.jpg';
+import logo from './images/logo.jpg';
 
 const TopBar = () => {
     const title = 'What Do You Want To Eat?';
```

Comment:
I think this const title should be changed to Flavor Finder (since it's the name of your app)

Reply:
Done.